bindings_test.go

```diff
@@ -331,6 +331,14 @@ func TestQuery(t *testing.T) {
 		"2",
 	})
 
+	// test predicate match
+	testCaptures(t, js, `((sum left: _ @left) (#match? @left "^[0-9]+$"))`, []string{
+		"1",
+	})
+
+	// test predicate not match
+	testCaptures(t, js, `((sum left: _ @left) (#not-match? @left "^[0-9]+$"))`, []string{})
```

Comment:
In particular, the original implementation in https://github.com/smacker/go-tree-sitter/pull/83 would fail this test, it would return `[]string{"1"}`.